<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title>publications | Danielle Denisko</title> <meta name="author" content="Danielle Denisko"> <meta name="description" content="Publications by categories in reversed chronological order. Generated by jekyll-scholar."> <meta name="keywords" content="bioinformatics, data science, jekyll, jekyll-theme, academic-website, portfolio-website"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="https://unpkg.com/bootstrap-table@1.22.1/dist/bootstrap-table.min.css"> <link rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons"> <link rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?19f3075a2d19613090fe9e16b564e1fe" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%E2%9A%9B%EF%B8%8F&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://dcdxy.github.io/publications/"> <link rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?e74e74bf055e5729d44a7d031a5ca6a5" media="none" id="highlight_theme_dark"> <script src="/assets/js/theme.js?6185d15ea1982787ad7f435576553d64"></script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"><span class="font-weight-bold">Danielle </span>Denisko</a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about</a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">publications<span class="sr-only">(current)</span></a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">cv</a> </li> <li class="nav-item "> <a class="nav-link" href="/teaching/">teaching</a> </li> <li class="nav-item "> <a class="nav-link" href="/bio/">bio</a> </li> <li class="nav-item "> <a class="nav-link" href="/=%5E/%5E=/">=^..^=</a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fa-solid fa-moon"></i> <i class="fa-solid fa-sun"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5"> <div class="post"> <header class="post-header"> <h1 class="post-title">publications</h1> <p class="post-description">Publications by categories in reversed chronological order. Generated by jekyll-scholar.</p> </header> <article> <div class="publications"> <h2 class="bibliography">2024</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" srcset=" /assets/img/publication_preview/black-cat-480.webp 480w, /assets/img/publication_preview/black-cat-800.webp 800w, /assets/img/publication_preview/black-cat-1400.webp 1400w, " sizes="200px" type="image/webp"></source> <img src="/assets/img/publication_preview/black-cat.gif" class="preview z-depth-1 rounded" width="100%" height="auto" alt="black-cat.gif" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="Park2024" class="col-sm-8"> <div class="title">Genomic data processing with GenomeFlow</div> <div class="author"> Junseok Park, Eduardo A Maury, Changhoon Oh, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Shin Donghoon, Danielle Denisko, Eunjung Alice Lee' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>BMC Bioinformatics</em>, 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="abstract hidden"> <p>Advances in genome sequencing technologies generate massive amounts of sequence data that are increasingly analyzed and shared through public repositories. On-demand infrastructure services on cloud computing platforms enable the processing of such large-scale genomic sequence data in distributed processing environments with a significant reduction in analysis time. However, parallel processing on cloud computing platforms presents many challenges to researchers, even skillful bioinformaticians. In particular, it is difficult to design a computing architecture optimized to reduce the cost of computing and disk storage as genomic data analysis pipelines often employ many heterogeneous tools with different resource requirements. To address these issues, we developed GenomeFlow, a tool for automated development of computing architecture and resource optimization on Google Cloud Platform, which allows users to process a large number of samples at minimal cost. We outline multiple use cases of GenomeFlow demonstrating its utility to significantly reduce computing time and cost associated with analyzing genomic and transcriptomic data from hundreds to tens of thousands of samples from several consortia. Here, we describe a step-by-step protocol on how to use GenomeFlow for a common genomic data processing task. We introduce this example protocol geared toward a bioinformatician with little experience in cloud computing and large data processing and estimate that it will take &lt;1 hour to execute.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Park2024</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Park, Junseok and Maury, Eduardo A and Oh, Changhoon and Donghoon, Shin and Denisko, Danielle and Lee, Eunjung Alice}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Genomic data processing with GenomeFlow}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{BMC Bioinformatics}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" srcset=" /assets/img/publication_preview/ponyo-sleeping-480.webp 480w, /assets/img/publication_preview/ponyo-sleeping-800.webp 800w, /assets/img/publication_preview/ponyo-sleeping-1400.webp 1400w, " sizes="200px" type="image/webp"></source> <img src="/assets/img/publication_preview/ponyo-sleeping.gif" class="preview z-depth-1 rounded" width="100%" height="auto" alt="ponyo-sleeping.gif" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="HCMV" class="col-sm-8"> <div class="title">Human cytomegalovirus harnesses host L1 retrotransposon for efficient replication</div> <div class="author"> Sung-Yeon Hwang*, Hyewon Kim*, <em>Danielle Denisko*</em>, and <span class="more-authors" title="click to view 9 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '9 more authors' ? 'Boxun Zhao*, Dohoon Lee, Jiseok Jeong, Jinuk Kim, Kiwon Park, Hee-Jung Choi, Sun Kim, Eunjung Alice Lee, Kwangseog Ahn' : '9 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">9 more authors</span> </div> <div class="periodical"> <em></em> 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="abstract hidden"> <p>Genetic parasites, including viruses and transposons, exploit components from the host for their own replication. However, little is known about virus-transposon interactions within host cells. Here, we discover a strategy where human cytomegalovirus (HCMV) hijacks L1 retrotransposon encoded protein during its replication cycle. HCMV infection upregulates L1 expression by enhancing both the expression of L1-activating transcription factors, YY1 and RUNX3 and the chromatin accessibility of L1 promoter regions. Increased L1 expression in turn promotes HCMV replicative fitness. Affinity proteomics reveals UL44, HCMV DNA polymerase subunit, as the most abundant viral binding protein of the L1 ribonucleoprotein (RNP) complex. UL44 directly interacts with L1 ORF2p, inducing DNA damage responses in replicating HCMV compartments. While increased L1- induced mutagenesis is not observed in HCMV for genetic adaptation, the interplay between UL44 and ORF2p accelerates viral DNA replication by resolving stalled replication forks. Our findings shed light on how HCMV exploits host retrotransposons for enhanced viral fitness.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">HCMV</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Hwang*, Sung-Yeon and Kim*, Hyewon and Denisko*, Danielle and Zhao*, Boxun and Lee, Dohoon and Jeong, Jiseok and Kim, Jinuk and Park, Kiwon and Choi, Hee-Jung and Kim, Sun and Lee, Eunjung Alice and Ahn, Kwangseog}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Human cytomegalovirus harnesses host L1 retrotransposon for efficient replication}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> </ol> <h2 class="bibliography">2023</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" srcset=" /assets/img/publication_preview/susuwatari-sugar-480.webp 480w, /assets/img/publication_preview/susuwatari-sugar-800.webp 800w, /assets/img/publication_preview/susuwatari-sugar-1400.webp 1400w, " sizes="200px" type="image/webp"></source> <img src="/assets/img/publication_preview/susuwatari-sugar.gif" class="preview z-depth-1 rounded" width="100%" height="auto" alt="susuwatari-sugar.gif" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="peaKO" class="col-sm-8"> <div class="title">Motif elucidation in ChIP-seq datasets with a knockout control</div> <div class="author"> <em>Danielle Denisko</em>, Coby Viner, and Michael M Hoffman</div> <div class="periodical"> <em>Bioinformatics Advances</em>, 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://doi.org/10.1093/bioadv/vbad031" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="abstract hidden"> <p>Chromatin immunoprecipitation-sequencing is widely used to find transcription factor binding sites, but suffers from various sources of noise. Knocking out the target factor mitigates noise by acting as a negative control. Paired wild-type and knockout (KO) experiments can generate improved motifs but require optimal differential analysis. We introduce peaKO—a computational method to automatically optimize motif analyses with KO controls, which we compare to two other methods. PeaKO often improves elucidation of the target factor and highlights the benefits of KO controls, which far outperform input controls.PeaKO is freely available at https://peako.hoffmanlab.org.michael.hoffman@utoronto.ca</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">peaKO</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Denisko, Danielle and Viner, Coby and Hoffman, Michael M}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Motif elucidation in ChIP-seq datasets with a knockout control}}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Bioinformatics Advances}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{3}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{vbad031}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{2635-0041}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1093/bioadv/vbad031}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1093/bioadv/vbad031}</span><span class="p">,</span>
  <span class="na">eprint</span> <span class="p">=</span> <span class="s">{https://academic.oup.com/bioinformaticsadvances/article-pdf/3/1/vbad031/49761827/vbad031.pdf}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li></ol> <h2 class="bibliography">2022</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" srcset=" /assets/img/publication_preview/bookmark-tabs-objects-480.webp 480w, /assets/img/publication_preview/bookmark-tabs-objects-800.webp 800w, /assets/img/publication_preview/bookmark-tabs-objects-1400.webp 1400w, " sizes="200px" type="image/webp"></source> <img src="/assets/img/publication_preview/bookmark-tabs-objects.gif" class="preview z-depth-1 rounded" width="100%" height="auto" alt="bookmark-tabs-objects.gif" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="BED" class="col-sm-8"> <div class="title">Assessing and assuring interoperability of a genomics file format</div> <div class="author"> Yi Nian Niu, Eric G Roberts, <em>Danielle Denisko</em>, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Michael M Hoffman' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>Bioinformatics</em>, 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://doi.org/10.1093/bioinformatics/btac327" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="abstract hidden"> <p>Bioinformatics software tools operate largely through the use of specialized genomics file formats. Often these formats lack formal specification, making it difficult or impossible for the creators of these tools to robustly test them for correct handling of input and output. This causes problems in interoperability between different tools that, at best, wastes time and frustrates users. At worst, interoperability issues could lead to undetected errors in scientific results.We developed a new verification system, Acidbio, which tests for correct behavior in bioinformatics software packages. We crafted tests to unify correct behavior when tools encounter various edge cases—potentially unexpected inputs that exemplify the limits of the format. To analyze the performance of existing software, we tested the input validation of 80 Bioconda packages that parsed the Browser Extensible Data (BED) format. We also used a fuzzing approach to automatically perform additional testing. Of 80 software packages examined, 75 achieved less than 70\% correctness on our test suite. We categorized multiple root causes for the poor performance of different types of software. Fuzzing detected other errors that the manually designed test suite could not. We also created a badge system that developers can use to indicate more precisely which BED variants their software accepts and to advertise the software’s performance on the test suite.Acidbio is available at https://github.com/hoffmangroup/acidbio.Supplementary data are available at Bioinformatics online.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">BED</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Niu, Yi Nian and Roberts, Eric G and Denisko, Danielle and Hoffman, Michael M}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Assessing and assuring interoperability of a genomics file format}}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Bioinformatics}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{38}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{13}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{3327-3336}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2022}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1367-4803}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1093/bioinformatics/btac327}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1093/bioinformatics/btac327}</span><span class="p">,</span>
  <span class="na">eprint</span> <span class="p">=</span> <span class="s">{https://academic.oup.com/bioinformatics/article-pdf/38/13/3327/49884326/btac327.pdf}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li></ol> <h2 class="bibliography">2021</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" srcset=" /assets/img/publication_preview/smiling-cat-typing-480.webp 480w, /assets/img/publication_preview/smiling-cat-typing-800.webp 800w, /assets/img/publication_preview/smiling-cat-typing-1400.webp 1400w, " sizes="200px" type="image/webp"></source> <img src="/assets/img/publication_preview/smiling-cat-typing.gif" class="preview z-depth-1 rounded" width="100%" height="auto" alt="smiling-cat-typing.gif" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="GA4GH" class="col-sm-8"> <div class="title">GA4GH: International policies and standards for data sharing across genomic research and healthcare</div> <div class="author"> Heidi L. Rehm, Angela J.H. Page, Lindsay Smith, and <span class="more-authors" title="click to view 199 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '199 more authors' ? 'Jeremy B. Adams, Gil Alterovitz, Lawrence J. Babb, Maxmillian P. Barkley, Michael Baudis, Michael J.S. Beauvais, Tim Beck, Jacques S. Beckmann, Sergi Beltran, David Bernick, Alexander Bernier, James K. Bonfield, Tiffany F. Boughtwood, Guillaume Bourque, Sarion R. Bowers, Anthony J. Brookes, Michael Brudno, Matthew H. Brush, David Bujold, Tony Burdett, Orion J. Buske, Moran N. Cabili, Daniel L. Cameron, Robert J. Carroll, Esmeralda Casas-Silva, Debyani Chakravarty, Bimal P. Chaudhari, Shu Hui Chen, J. Michael Cherry, Justina Chung, Melissa Cline, Hayley L. Clissold, Robert M. Cook-Deegan, Mélanie Courtot, Fiona Cunningham, Miro Cupak, Robert M. Davies, Danielle Denisko, Megan J. Doerr, Lena I. Dolman, Edward S. Dove, L. Jonathan Dursi, Stephanie O.M. Dyke, James A. Eddy, Karen Eilbeck, Kyle P. Ellrott, Susan Fairley, Khalid A. Fakhro, Helen V. Firth, Michael S. Fitzsimons, Marc Fiume, Paul Flicek, Ian M. Fore, Mallory A. Freeberg, Robert R. Freimuth, Lauren A. Fromont, Jonathan Fuerth, Clara L. Gaff, Weiniu Gan, Elena M. Ghanaim, David Glazer, Robert C. Green, Malachi Griffith, Obi L. Griffith, Robert L. Grossman, Tudor Groza, Jaime M. Guidry Auvil, Roderic Guigó, Dipayan Gupta, Melissa A. Haendel, Ada Hamosh, David P. Hansen, Reece K. Hart, Dean Mitchell Hartley, David Haussler, Rachele M. Hendricks-Sturrup, Calvin W.L. Ho, Ashley E. Hobb, Michael M. Hoffman, Oliver M. Hofmann, Petr Holub, Jacob Shujui Hsu, Jean-Pierre Hubaux, Sarah E. Hunt, Ammar Husami, Julius O. Jacobsen, Saumya S. Jamuar, Elizabeth L. Janes, Francis Jeanson, Aina Jené, Amber L. Johns, Yann Joly, Steven J.M. Jones, Alexander Kanitz, Kazuto Kato, Thomas M. Keane, Kristina Kekesi-Lafrance, Jerome Kelleher, Giselle Kerry, Seik-Soon Khor, Bartha M. Knoppers, Melissa A. Konopko, Kenjiro Kosaki, Martin Kuba, Jonathan Lawson, Rasko Leinonen, Stephanie Li, Michael F. Lin, Mikael Linden, Xianglin Liu, Isuru Udara Liyanage, Javier Lopez, Anneke M. Lucassen, Michael Lukowski, Alice L. Mann, John Marshall, Michele Mattioni, Alejandro Metke-Jimenez, Anna Middleton, Richard J. Milne, Fruzsina Molnár-Gábor, Nicola Mulder, Monica C. Munoz-Torres, Rishi Nag, Hidewaki Nakagawa, Jamal Nasir, Arcadi Navarro, Tristan H. Nelson, Ania Niewielska, Amy Nisselle, Jeffrey Niu, Tommi H. Nyrönen, Brian D. O’Connor, Sabine Oesterle, Soichi Ogishima, Vivian Ota Wang, Laura A.D. Paglione, Emilio Palumbo, Helen E. Parkinson, Anthony A. Philippakis, Angel D. Pizarro, Andreas Prlic, Jordi Rambla, Augusto Rendon, Renee A. Rider, Peter N. Robinson, Kurt W. Rodarmer, Laura Lyman Rodriguez, Alan F. Rubin, Manuel Rueda, Gregory A. Rushton, Rosalyn S. Ryan, Gary I. Saunders, Helen Schuilenburg, Torsten Schwede, Serena Scollen, Alexander Senf, Nathan C. Sheffield, Neerjah Skantharajah, Albert V. Smith, Heidi J. Sofia, Dylan Spalding, Amanda B. Spurdle, Zornitza Stark, Lincoln D. Stein, Makoto Suematsu, Patrick Tan, Jonathan A. Tedds, Alastair A. Thomson, Adrian Thorogood, Timothy L. Tickle, Katsushi Tokunaga, Juha Törnroos, David Torrents, Sean Upchurch, Alfonso Valencia, Roman Valls Guimera, Jessica Vamathevan, Susheel Varma, Danya F. Vears, Coby Viner, Craig Voisin, Alex H. Wagner, Susan E. Wallace, Brian P. Walsh, Marc S. Williams, Eva C. Winkler, Barbara J. Wold, Grant M. Wood, J. Patrick Woolley, Chisato Yamasaki, Andrew D. Yates, Christina K. Yung, Lyndon J. Zass, Ksenia Zaytseva, Junjun Zhang, Peter Goodhand, Kathryn North, Ewan Birney' : '199 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">199 more authors</span> </div> <div class="periodical"> <em>Cell Genomics</em>, 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://doi.org/10.1016/j.xgen.2021.100029" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="abstract hidden"> <p>The Global Alliance for Genomics and Health (GA4GH) aims to accelerate biomedical advances by enabling the responsible sharing of clinical and genomic data through both harmonized data aggregation and federated approaches. The decreasing cost of genomic sequencing (along with other genome-wide molecular assays) and increasing evidence of its clinical utility will soon drive the generation of sequence data from tens of millions of humans, with increasing levels of diversity. In this perspective, we present the GA4GH strategies for addressing the major challenges of this data revolution. We describe the GA4GH organization, which is fueled by the development efforts of eight Work Streams and informed by the needs of 24 Driver Projects and other key stakeholders. We present the GA4GH suite of secure, interoperable technical standards and policy frameworks and review the current status of standards, their relevance to key domains of research and clinical care, and future plans of GA4GH. Broad international participation in building, adopting, and deploying GA4GH standards and frameworks will catalyze an unprecedented effort in data sharing that will be critical to advancing genomic medicine and ensuring that all populations can access its benefits.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">GA4GH</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{GA4GH: International policies and standards for data sharing across genomic research and healthcare}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Rehm, {Heidi L.} and Page, {Angela J.H.} and Smith, Lindsay and Adams, {Jeremy B.} and Alterovitz, Gil and Babb, {Lawrence J.} and Barkley, {Maxmillian P.} and Baudis, Michael and Beauvais, {Michael J.S.} and Beck, Tim and Beckmann, {Jacques S.} and Beltran, Sergi and Bernick, David and Bernier, Alexander and Bonfield, {James K.} and Boughtwood, {Tiffany F.} and Bourque, Guillaume and Bowers, {Sarion R.} and Brookes, {Anthony J.} and Brudno, Michael and Brush, {Matthew H.} and Bujold, David and Burdett, Tony and Buske, {Orion J.} and Cabili, {Moran N.} and Cameron, {Daniel L.} and Carroll, {Robert J.} and Casas-Silva, Esmeralda and Chakravarty, Debyani and Chaudhari, {Bimal P.} and Chen, {Shu Hui} and Cherry, {J. Michael} and Chung, Justina and Cline, Melissa and Clissold, {Hayley L.} and Cook-Deegan, {Robert M.} and Courtot, M{\'e}lanie and Cunningham, Fiona and Cupak, Miro and Davies, {Robert M.} and Denisko, Danielle and Doerr, {Megan J.} and Dolman, {Lena I.} and Dove, {Edward S.} and Dursi, {L. Jonathan} and Dyke, {Stephanie O.M.} and Eddy, {James A.} and Eilbeck, Karen and Ellrott, {Kyle P.} and Fairley, Susan and Fakhro, {Khalid A.} and Firth, {Helen V.} and Fitzsimons, {Michael S.} and Fiume, Marc and Flicek, Paul and Fore, {Ian M.} and Freeberg, {Mallory A.} and Freimuth, {Robert R.} and Fromont, {Lauren A.} and Fuerth, Jonathan and Gaff, {Clara L.} and Gan, Weiniu and Ghanaim, {Elena M.} and Glazer, David and Green, {Robert C.} and Griffith, Malachi and Griffith, {Obi L.} and Grossman, {Robert L.} and Groza, Tudor and {Guidry Auvil}, {Jaime M.} and Guig{\'o}, Roderic and Gupta, Dipayan and Haendel, {Melissa A.} and Hamosh, Ada and Hansen, {David P.} and Hart, {Reece K.} and Hartley, {Dean Mitchell} and Haussler, David and Hendricks-Sturrup, {Rachele M.} and Ho, {Calvin W.L.} and Hobb, {Ashley E.} and Hoffman, {Michael M.} and Hofmann, {Oliver M.} and Holub, Petr and Hsu, {Jacob Shujui} and Hubaux, Jean-Pierre and Hunt, {Sarah E.} and Husami, Ammar and Jacobsen, {Julius O.} and Jamuar, {Saumya S.} and Janes, {Elizabeth L.} and Jeanson, Francis and Jen{\'e}, Aina and Johns, {Amber L.} and Joly, Yann and Jones, {Steven J.M.} and Kanitz, Alexander and Kato, Kazuto and Keane, {Thomas M.} and Kekesi-Lafrance, Kristina and Kelleher, Jerome and Kerry, Giselle and Khor, Seik-Soon and Knoppers, {Bartha M.} and Konopko, {Melissa A.} and Kosaki, Kenjiro and Kuba, Martin and Lawson, Jonathan and Leinonen, Rasko and Li, Stephanie and Lin, {Michael F.} and Linden, Mikael and Liu, Xianglin and Liyanage, {Isuru Udara} and Lopez, Javier and Lucassen, {Anneke M.} and Lukowski, Michael and Mann, {Alice L.} and Marshall, John and Mattioni, Michele and Metke-Jimenez, Alejandro and Middleton, Anna and Milne, {Richard J.} and Moln{\'a}r-G{\'a}bor, Fruzsina and Mulder, Nicola and Munoz-Torres, {Monica C.} and Nag, Rishi and Nakagawa, Hidewaki and Nasir, Jamal and Navarro, Arcadi and Nelson, {Tristan H.} and Niewielska, Ania and Nisselle, Amy and Niu, Jeffrey and Nyr{\"o}nen, {Tommi H.} and O{\textquoteright}Connor, {Brian D.} and Oesterle, Sabine and Ogishima, Soichi and {Ota Wang}, Vivian and Paglione, {Laura A.D.} and Palumbo, Emilio and Parkinson, {Helen E.} and Philippakis, {Anthony A.} and Pizarro, {Angel D.} and Prlic, Andreas and Rambla, Jordi and Rendon, Augusto and Rider, {Renee A.} and Robinson, {Peter N.} and Rodarmer, {Kurt W.} and Rodriguez, {Laura Lyman} and Rubin, {Alan F.} and Rueda, Manuel and Rushton, {Gregory A.} and Ryan, {Rosalyn S.} and Saunders, {Gary I.} and Schuilenburg, Helen and Schwede, Torsten and Scollen, Serena and Senf, Alexander and Sheffield, {Nathan C.} and Skantharajah, Neerjah and Smith, {Albert V.} and Sofia, {Heidi J.} and Spalding, Dylan and Spurdle, {Amanda B.} and Stark, Zornitza and Stein, {Lincoln D.} and Suematsu, Makoto and Tan, Patrick and Tedds, {Jonathan A.} and Thomson, {Alastair A.} and Thorogood, Adrian and Tickle, {Timothy L.} and Tokunaga, Katsushi and T{\"o}rnroos, Juha and Torrents, David and Upchurch, Sean and Valencia, Alfonso and Guimera, {Roman Valls} and Vamathevan, Jessica and Varma, Susheel and Vears, {Danya F.} and Viner, Coby and Voisin, Craig and Wagner, {Alex H.} and Wallace, {Susan E.} and Walsh, {Brian P.} and Williams, {Marc S.} and Winkler, {Eva C.} and Wold, {Barbara J.} and Wood, {Grant M.} and Woolley, {J. Patrick} and Yamasaki, Chisato and Yates, {Andrew D.} and Yung, {Christina K.} and Zass, {Lyndon J.} and Zaytseva, Ksenia and Zhang, Junjun and Goodhand, Peter and North, Kathryn and Birney, Ewan}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1016/j.xgen.2021.100029}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Cell Genomics}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{2666-979X}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Cell Press}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li></ol> <h2 class="bibliography">2018</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" srcset=" /assets/img/publication_preview/ghibli-totoro-forest-480.webp 480w, /assets/img/publication_preview/ghibli-totoro-forest-800.webp 800w, /assets/img/publication_preview/ghibli-totoro-forest-1400.webp 1400w, " sizes="200px" type="image/webp"></source> <img src="/assets/img/publication_preview/ghibli-totoro-forest.gif" class="preview z-depth-1 rounded" width="100%" height="auto" alt="ghibli-totoro-forest.gif" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="Denisko2018" class="col-sm-8"> <div class="title">Classification and interaction in random forests</div> <div class="author"> <em>Danielle Denisko</em>, and Michael M Hoffman</div> <div class="periodical"> <em>Proceedings of the National Academy of Sciences</em>, 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="http://www.pnas.org/content/115/8/1690.abstract" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Denisko2018</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Denisko, Danielle and Hoffman, Michael M}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1073/pnas.1800256115}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{0027-8424}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Proceedings of the National Academy of Sciences}</span><span class="p">,</span>
  <span class="na">mendeley-groups</span> <span class="p">=</span> <span class="s">{Commentary}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{8}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1690--1692}</span><span class="p">,</span>
  <span class="na">pmid</span> <span class="p">=</span> <span class="s">{29440440}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Classification and interaction in random forests}}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{http://www.pnas.org/content/115/8/1690.abstract}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{115}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2018}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li></ol> </div> </article> </div> </div> <footer class="fixed-bottom"> <div class="container mt-0"> © Copyright 2024 Danielle Denisko. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. Photos from <a href="https://unsplash.com" target="_blank" rel="external nofollow noopener">Unsplash</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.8/dist/medium-zoom.min.js" integrity="sha256-7PhEpEWEW0XXQ0k6kQrPKwuoIomz8R8IYyuU1Qew4P8=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?7b30caa5023af4af8408a472dc4e1ebb"></script> <script defer src="https://unpkg.com/bootstrap-table@1.22.1/dist/bootstrap-table.min.js"></script> <script src="/assets/js/no_defer.js?d633890033921b33e0ceb13d22340a9c"></script> <script defer src="/assets/js/common.js?acdb9690d7641b2f8d40529018c71a01"></script> <script defer src="/assets/js/copy_code.js?9b43d6e67ddc7c0855b1478ee4c48c2d" type="text/javascript"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> </body> </html>